pimd/pim_msdp.h

```diff
@@ -152,6 +152,9 @@ struct pim_msdp_peer {
 	char *acl_in;
 	/** SA output access list name. */
 	char *acl_out;
+
+	/** SA maximum amount. */
+	size_t sa_limit;
```

Comment:
since the limit is 4billion why not a uint32_t?

Reply:
Sure, makes sense.